go.mod

```diff
@@ -178,8 +180,8 @@ replace (
 	github.com/crewjam/saml => github.com/crewjam/saml v0.4.5
 	github.com/docker/distribution => github.com/docker/distribution v2.8.1+incompatible
 	github.com/docker/docker => github.com/docker/docker v20.10.17+incompatible
-	github.com/rancher/rancher/pkg/apis => github.com/rancher/rancher/pkg/apis v0.0.0-20230512212658-eaa7b90049a4
-	github.com/rancher/rancher/pkg/client => github.com/rancher/rancher/pkg/client v0.0.0-20230512212658-eaa7b90049a4
+	github.com/rancher/rancher/pkg/apis => github.com/rancher/rancher/pkg/apis v0.0.0-20230901132600-5e1ee2611616
+	github.com/rancher/rancher/pkg/client => github.com/rancher/rancher/pkg/client v0.0.0-20230901132600-5e1ee2611616
```

Comment:
It's not clear what version of rancher/rancher to reference yet.

Reply:
Right now it's pointing to a merge commit [5e1ee2611616](https://github.com/rancher/rancher/commit/5e1ee2611616) where `inheritedClusterRoles` was introduced.

Reply:
@a-blender Is this acceptable?

Reply:
If TF rancher2 requires 'inheritedClusterRoles' from the rancher backend for this feature to work correctly, than yes but in general the provider should reference the latest commit on rancher release/v2.8. Why are we updating those other deps? Are they indirect deps? I will need to formally review this later.

Reply:
Yes, this requires `inheritedClusterRoles` from the rancher backend.

> Why are we updating those other deps? Are they indirect?
We don't, the Go toolchain does when we point `replace`s to the above-mentioned commit and run `go mod tidy`.

Reply:
On second thought, I think that is the safest option. Lgtm